lib/Sema/CSApply.cpp

```diff
@@ -2443,82 +2443,111 @@ namespace {
         // single argument case.
         assert(cs.getType(expr->getArg())->isEqual(stringTy) &&
                "single argument should be string");
-        return expr;
-      }
-
-      for (unsigned i = 0, e = args->getNumElements(); i != e; ++i) {
-        auto argExpr = args->getElement(i);
-        auto argType = cs.getType(argExpr);
-
-        // CSGen has already ensured that the first argument is a string, so skip
-        // the first argument.
-        if (i == 0) {
-          assert(argType->isEqual(stringTy) &&
-                 "first argument should be string");
-          continue;
-        }
+      } else {
```

Comment:
fyi, all the changes until L2521 are just putting existing code in the else block and indenting it

Reply:
Thanks for the helpful note!